en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -860,6 +860,31 @@ result set.
 
 .. end-filtering
 
+Retrieving data from belongsToMany associations
+===================================================
+
+Sometimes you need to retrieve data from a associative table. 
```

Comment:
We generally use the term 'junction' when referring to the joining table is a belongs to many association. It would be good to be consistent here.

Reply:
is a => in a

Reply:
You are right @markstory ! What statement do you suggest here to explain about _joinData and junction()?